src/styles/reset.css

```diff
@@ -0,0 +1,29 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: none;
+}

```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document
Same for border. a small amount of elements have it by default. remove only where needed